libraries/AP_HAL_ChibiOS/hwdef/MatekL431-Rangefinder/hwdef.dat

```diff
@@ -6,11 +6,9 @@ define HAL_DISABLE_ADC_DRIVER TRUE
 
 # support all rangefinder types
 define HAL_PERIPH_ENABLE_RANGEFINDER
+define RANGEFINDER_MAX_INSTANCES 2
 
 define AP_PERIPH_RANGEFINDER_PORT_DEFAULT 2
 
 # setup for MSP
 define HAL_MSP_ENABLED 1
-
-# some lidars take a long time to init, keep probing till we find it
```

Comment:
this will cause problems on nodes with separate power on the lidar, which is quite common on complex vehicles.
ideally we would problem always with a rangefinder.reinit() call, but only when disarmed, and have a DEBUG option bit to disable the probing after boot

Reply:
Fixed as discussed.  We continue to prop continuous but only when disarmed.  We now also have an options param to switch prop continuous off if users want.